app/src/pages/inside/stepPage/modals/makeDecisionModal/machineLearningSuggestions/machineLearningSuggestions.jsx

```diff
@@ -127,8 +124,9 @@ MachineLearningSuggestions.propTypes = {
   setModalState: PropTypes.func.isRequired,
   itemData: PropTypes.object,
   collapseTabsExceptCurr: PropTypes.func.isRequired,
-  isNarrowView: PropTypes.bool,
 };
 MachineLearningSuggestions.defaultProps = {
   items: [],
+  itemData: {},
+  collapseTabsExceptCurr: () => {},
```

Comment:
It is redundant in defaultProps, because `collapseTabsExceptCurr` is obligatory field